package/system/ubox/files/log.init

```diff
@@ -53,7 +53,9 @@ start_service_file()
 	[ -z "${log_file}" ] && return
 
 	[ "$_BOOT" = "1" ] &&
-		[ "$(procd_get_mountpoints "${log_file}")" ] && return 0
+		[ "$(procd_get_mountpoints "${log_file}")" ] &&
+		! grep -q ".* $(procd_get_mountpoints "${log_file}") " /proc/mounts &&
```

Comment:
Nit: Please assign `$(procd_get_mountpoints "${log_file}")` to a local variable and use that instead of calling `procd_get_mountpoints` twice.

Reply:
Done. Thank you.